### DISCUSSION TITLE
The Application "vibook" can't be opened.

### DISCUSSION THREAD

**Discussion by randiekas** [tauri-apps/tauri#7817] [Q&A]
<img width="850" alt="image" src="https://github.com/tauri-apps/tauri/assets/5701058/55b17cc0-61fd-4202-bae7-0ca569d33d8b">

I would like to report an issue I am experiencing with my own app  "vibook" application. The application runs smoothly on my laptop without any problems. However, when I try to install it on another laptop, I receive the following error message:

Error Message: The Application "vibook" can't be opened.

I have tried several steps, such as uninstalling and reinstalling the application, but this error message continues to appear. I have also ensured that the other laptop meets the necessary specifications and runs a compatible operating system.

I would appreciate assistance in resolving this issue so that I can use the "vibook" application effectively on the other laptop. Thank you!

yarn tauri info:

<img width="939" alt="image" src="https://github.com/tauri-apps/tauri/assets/5701058/8b9a0555-10cc-4d0e-81f9-2231c9836ff0">

**Comment by JonasKruckenberg** [comment]
Is your app signed?

**Reply by randiekas** [reply]
i'm new in tauri, can you explain how to do signed step by step  ?

**Reply by JonasKruckenberg** [reply]
Yeah alright, so the issue on macOS is that whenever you build an app *on your device* apple then prevents that app from running on *other devices* by default. Part of their security (or money making scheme 🤷🏻‍♂️) regardless, that's the world we live in.

Now, to be able to run your app on another device you have a couple options:
- Control-click the app and then select "open" from the menu
- Actually code sign your app like you should, there is a guide as to how to set the up with Tauri [here](https://tauri.app/v1/guides/distribution/sign-macos)

Edit: Have you checked that both your laptops are the same arch? (e.g. both have an Intel chip or both have an Apple Silicon chip? Mixing those won't work)
